src/Provider/PointCloudProvider.js

```diff
@@ -255,7 +253,10 @@ export default {
             points.tightbbox.max.z *= layer.metadata.scale;
             points.tightbbox.translate(node.bbox.min);
             points.material.transparent = layer.opacity < 1.0;
-            points.material.uniforms.opacity.value = layer.opacity;
+            points.material.opacity = layer.opacity;
+            if (points.material.refreshUniforms) {
+                points.onBeforeRender = points.material.refreshUniforms.bind(points.material);
```

Comment:
Just a comment here: I'd prefer to keep the old mechanism (update during process).

That way the `onBeforeRender` callbacks provided by `three` are still available for the user.